pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -91,19 +92,23 @@ func PrintDownStreamVersions(output io.Writer, minSupportedVersion string) {
 }
 
 func PrintUpStreamVersions(output io.Writer, minSupportedVersion string) {
-	resp, err := getResponseBody("https://registry.hub.docker.com/v2/repositories/openshift/origin/tags/")
+	dockerRegistryUrl := "https://registry.hub.docker.com/v2/repositories/openshift/origin/tags/?page_size=1024"
+	resp, err := getResponseBody(dockerRegistryUrl)
 	if err != nil {
 		fmt.Fprintf(output, "Error Occured %s", err)
 		return
 	}
 	defer resp.Body.Close()
 	decoder := json.NewDecoder(resp.Body)
 	var data ImageTags
-	err = decoder.Decode(&data)
-	if err != nil {
-		fmt.Printf("%T\n%s\n%#v\n", err, err, err)
-		return
-	}
```

Comment:
@LalatenduMohanty You are un-marshalling into a provided data structure - https://github.com/minishift/minishift/blob/29c6b76418c372882f00b29461b7d75195aa1464/pkg/minishift/openshift/version/openshift_versions.go#L101-L101 (ImageTags). You will need to change this to match the format of the v1 output.